clash-ghc/src-bin-common/Clash/GHCi/Common.hs

```diff
@@ -17,11 +16,7 @@ import           Clash.Netlist.Types    (TopEntityT(..))
 
 -- The GHC interface
 import qualified DynFlags
-#if MIN_VERSION_base(4,11,0)
```

Comment:
You should update the lower bound of the base dependency in clash-ghc.cabal

Reply:
Good idea